gnucash/report/options-utilities.scm

```diff
@@ -227,29 +217,24 @@
     ;; usually the option name is: (N_ "Parent account balances")
     optname-parent-balance-mode
     (string-append sort-tag "a")
-    (N_ "How to show the balances of parent accounts.")
+    (N_ "How to show the balances of parent accounts.\n\
+Account Balance in the parent account, excluding any subaccounts.\n\
```

Comment:
If you want to list descriptions of the possible options in the widget level tooltip, it would help if they are presented in a bullet list. Goes for other options further down as well.

Reply:
I am not sure I like this, if I could think of shorter appropriate text for the longest item of the option I would of used the tool tips, in this case...
'Calculate the subtotal for this parent account and all of its subaccounts, and show this as the parent account balance.'
and the one below...
'Show subtotals for selected parent accounts which have subaccounts'

Reply:
I'm not sure I can follow your train of thought here so I can't tell if you understood what I suggested.

I'll try again - I think you have done the right thing to keep the short multi-choice names here (Account Balance, Calculate Subtotal, Do not show). However you have also chosen to add the longer descriptions in the common widget tooltip. In there the 3 descriptions are added just one after the other. It's not that clear these descriptions refer to 3 possible choices. So I would suggest to at least do this:

```
(N_ "How to show the balances of parent accounts.\n\
* Account Balance in the parent account, excluding any subaccounts.\n\
* Calculate the subtotal for this parent account and all of its subaccounts, and show this as the parent account balance.\n\
* Do not show any balances of parent accounts.")
```